src/borg/platform/base.py

```diff
@@ -1,5 +1,19 @@
 import os
 
+from .. import platform
```

Comment:
If it really isn't supposed to work then I'd make a lazy intermediary that imports once-on-demand.